CONTRIBUTING.md

```diff
@@ -25,44 +84,79 @@ source venv/bin/activate
 ```
 
 ### Pre-commit
-We also provide a simple pre-commit hook for git that prevents you from commiting unchecked code. Note that this action will reinitialize the git repository inside the project directory, if you have already created one. To use it, run
+We also provide a simple pre-commit hook for `git` that prevents you from commiting unchecked code. Note that this action will reinitialize the git repository inside the project directory, if you have already created one. To use it, run
```

Comment:
I do not understand, it create a new hook,  not `reinitialize the git repository`